src/main/kotlin/demo/dto/application/ApplicationPayloads.kt

```diff
@@ -53,4 +59,30 @@ data class ApplicationNoUserPayload(
             quizzesResults = Collections.emptySet(),
             interviews = Collections.emptySet(),
     )
+}
+
+////////
+//// DTO TO READ APPLICATIONS
```

Comment:
Czysto teoretycznie nie lepiej to ApplicationDTO wyrzucić do pliku z taką nazwą i nie będzie potrzeby na taki komentarz? Chyba że taką mamy konwencję że trzymamy obok siebie DTO i payload.

Reply:
Nie, to co mówisz jest jak najbardziej ok, powinniśmy odpowiednie DTO trzymać w odpowiednich miejscach. Na razie zdecydowaliśmy tylko, że robimy tak, żeby było, a po approve'ach prowadzących zrobimy tak, żeby było ładnie. Ogólnie mam pomysł na strukturę katalogów i co gdzie ma być, ale to Wam wyjaśnię kiedyś i obgadamy, czy na takie coś się zgadzamy.

Reply:
Podczas wstawiania tego kodu została przyjęta strategia "na pałę"